pkgs/development/python-modules/pgsanity/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, postgresql }:
+
+buildPythonPackage rec {
+  pname = "pgsanity";
+  version = "0.2.9";
+  rev = "3bd391be1c5f0e2ce041652bdaf1d6b54424c6d8";
+
+  src = fetchFromGitHub {
+    owner = "markdrago";
+    repo = "pgsanity";
+    sha256 = "1imdgzh0nyq43w41wr62wx25wg45fjz3azwybsla81lxxp4rlvjg";
+    inherit rev;
+  };
+
+  doCheck = false;
+    
+  propagatedBuildInputs = [ postgresql ];
+
+  meta = {
+    homepage = "http://github.com/markdrago/pgsanity";
+    description = "PgSanity checks the syntax of Postgresql SQL files";
+    longDescription = ''
+      PgSanity checks the syntax of Postgresql SQL files by taking a file that has a list of bare SQL in it, making that file look like a C file with embedded SQL, run it through ecpg and let ecpg report on the syntax errors of the SQL.
+    '';
+    license = stdenv.lib.licenses.mit;
```

Comment:
It's not a hard rule, but are there any reasons you don't want to take maintainership for this package?

Reply:
I will be happy to take up that role! I was just unsure whether as a newcomer to the community it was okay to do that. :)

Reply:
Package maintainers are mostly people to ping if their package(s) break and other contributors are unsure why. As most of the new contributors (including me when I started doing Nix) added themselves to the maintainers list in `maintainers/maintainer-list.nix` this should be absolutely ok in your case :)